app/controllers/v0/institution_programs_controller.rb

```diff
@@ -9,7 +9,9 @@ def autocomplete
       @data = []
       if params[:term].present?
         @search_term = params[:term]&.strip&.downcase
-        @data = InstitutionProgram.version(@version[:number]).autocomplete(@search_term)
+        @data = InstitutionProgram.joins('INNER JOIN versions v ON v.id = institutions.version_id')
+                                  .where('v.number = ?', @version.number)
+                                  .autocomplete(@search_term)
```

Comment:
Why make this change?  Are you planning to remove `version` from `institution_programs`?

Reply:
@jholton by leveraging the `version_id` of the parent institution, `version` is no longer really necessary for `institution_programs`. This would also be the case for `school_certifying_officials`, which has a similar relationship to `institution`. 

It seems that `version` could be removed from both, but I wasn't entirely sure if it would be a good idea to add a migration for its removal.

Reply:
Makes sense.  I think it's a good idea to remove the column when we're sure it's not being used anywhere.

Reply:
I agree, it probably is best to go ahead and add a migration to remove that unused column. I will also have to do some clean up on the unit tests which use version there.

Reply:
Hi @jholton, after discussing this with some of the folks on my team, we decided it would be best to go ahead and plan to do this in a separate issue where we would also remove the other deprecated `version` columns.

Reply:
Sounds good to me.